staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -128,8 +129,8 @@ func newApfServerWithSingleRequest(t *testing.T, decision mockDecision) *httptes
 			t.Errorf("execute should not be invoked")
 		}
 		// atomicReadOnlyExecuting can be either 0 or 1 as we test one request at a time.
-		if decision != decisionSkipFilter && atomicReadOnlyExecuting != 1 {
-			t.Errorf("Wanted %d requests executing, got %d", 1, atomicReadOnlyExecuting)
+		if decision != decisionSkipFilter && atomic.LoadInt32(&atomicReadOnlyExecuting) != 1 {
+			t.Errorf("Wanted %d requests executing, got %d", 1, atomic.LoadInt32(&atomicReadOnlyExecuting))
```

Comment:
I recommend saving it in a variable, because technically the value could change in between and could lead to inconsistent behavior:
```
executing := atomic.LoadInt32(&atomicReadOnlyExecuting)
if decision != decisionSkipFilter && executing != 1 {
    t.Errorf("Wanted %d requests executing, got %d", 1, executing)
```

Reply:
Done